source/messaging/extending-messaging-with-integrations.md

```diff
@@ -1,9 +1,111 @@
+
+#### Jira Plugin
+
+- Two-way integration between Mattermost and Jira, developed by Mattermost.
+- Send event notifications from your Jira projects to Mattermost channels, with full JQL filtering support.
+- Create and transition Jira issues, and attach Mattermost messages to Jira in the Mattermost user interface.
+- Supports Jira Core and Jira Software products, for Cloud, Server and Data Center platforms.
+- Source code + docs: https://github.com/mattermost/mattermost-plugin-jira
+
+### Productivity integrations
+
+There are several ways to connect your productivity tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### Google Calendar
+
+ - Time management tool for Google Calendar, developed by Wasim Thabraze.
+ - Get reminders about appointments and meetings from a configured Google Calendar to any Mattermost channel.
+ - Source code + docs: https://github.com/waseem18/mattermost-plugin-google-calendar
+
+#### Remind Plugin
+
+ - A Mattermost plugin that sets reminders for users and channels, developed by Scott Lee Davis.
+ - Set reminders for one-time or recurring events and tasks in any Mattermost channel.
+ - Source code + docs: https://github.com/scottleedavis/mattermost-plugin-remind
+
+### Voice, video, and screensharing integrations
+
+There are several ways to work with voice, video, and screensharing in Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### Zoom
+
+- Start and join voice calls, video calls, and use screensharing with your team members, developed by Mattermost.
+- Click a video icon in a Mattermost channel to invite team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
+- Requires one paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret.
+- Supports a self-hosted cloud solution and a vendor-hosted cloud solution.
+- Docs available at: https://docs.mattermost.com/integrations/zoom.html.
+- Source code available at: https://github.com/mattermost/mattermost-plugin-zoom.
+
+#### Microsoft Teams Meetings
+
+- Start and join voice calls, video calls, and use screensharing with your team members, developed by Mattermost.
+- Click a video icon in a Mattermost channel to invite team members to join a Microsoft Teams Meeting, hosted using the credentials of the user who initiated the call.
+- Requires a Microsoft Teams account.
+- Requires Mattermost Enterprise Edition E20.
+- Source code and docs available at: https://github.com/mattermost/mattermost-plugin-msteams-meetings.
+
 ## Incoming Webhooks
 
 Incoming webhooks from external integrations can post messages to Mattermost in public and private channels. Learn more about setting up incoming webhooks on our [documentation page](https://docs.mattermost.com/developer/webhooks-incoming.html).
```

Comment:
I believe these webhook links are now outdated (this one and the one below), as well as the slash command one

Reply:
They are! I'll update.